src/mame/netronics/exp85.cpp

```diff
@@ -80,21 +105,38 @@ class exp85_state : public driver_device
 	int sid_r();
 	void sod_w(int state);
 
+	/* memory mapping during reset and interrupt mgmt */
+	void status_out(u8 status);
+	bool prep_int_call = false;
```

Comment:
Modern MAME style preference is that all member variables have an `m_` prefix to make it clear what's going on, and that member variables and member functions be grouped separately.  Usually all the functions and then all of the variables.

I realize pretty much everything I've discussed so far is inherited technical debt rather than your new work.  That's the occupational hazard of picking up a skeleton driver that's 15 years old.  Cleaning all that up now will make things simpler when the RAM and FDC are added to run CP/M later.

Reply:
Fixed